### DISCUSSION TITLE
Use available space better on small width

### DISCUSSION THREAD

**Discussion by tpetry** [spatie/ray#396] [Ideas]
When debugging a simple Array for example the available space gets used very efficiently:
<img width="399" alt="Bildschirmfoto 2021-03-29 um 12 32 08" src="https://user-images.githubusercontent.com/315686/112824438-d0543c00-908a-11eb-9285-00c049a7c745.png">

But when displaying objects too much whitespace is used on the left to have left-aligned labels:
<img width="399" alt="Bildschirmfoto 2021-03-29 um 12 34 30" src="https://user-images.githubusercontent.com/315686/112824677-20cb9980-908b-11eb-92fd-a83879c9a0c2.png">

Wouldn't it be more space efficient to move the labels (Class Name, Attributes) on top of their values so the values are better visible for small window widths? I need to scroll all the time for seeing one or two level nested structures because 29% of the space is "lost" on my example screen.

**Comment by freekmurze** [comment]
@willemvb could you take a look at this?

**Reply by willemvb** [reply]
I added a PR to improve on this, awaiting @AdrianMrn's stamp of approval!

**Reply by AdrianMrn** [reply]
Looks great! @tpetry this will be included in the next patch (will launch sometime this week): 

![image](https://user-images.githubusercontent.com/12762044/114366265-92bbdc80-9b7b-11eb-97fc-418b572fb898.png)

Thanks for the suggestion, keep 'em coming!
